app/javascript/packs/hello_react.jsx

```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
```

Comment:
Is this file needed?

Reply:
@crowesn  No.  I think it came over with the webpacker/reactjs generator.  Let me remove it and send the PR back up.

Reply:
Removed : https://github.com/uclibs/uc_drc/pull/93